app/mailers/user_mailer.rb

```diff
@@ -248,6 +248,20 @@ def account_verified(profile:)
     end
   end
 
+  def idv_please_call(**)
```

Comment:
`**` is there to support backwards compat with `in_person_please_call`, can be cleaned up later